src/css/map-view.css

```diff
@@ -1365,15 +1365,19 @@ other class: .ui-slider-range */
   width: 100%;
 }
 
+.viewfinder__search-input {
+  margin: 0 24px;
+}
+
 .viewfinder-predictions {
   list-style: none;
-  margin: 0;
+  margin: -16px 24px 0 24px;
```

Comment:
Instead of negative margins, I was planning 

```
.viewfinder__predictions,
.viewfinder__search-input {
  margin: 0 24px;
}
```

Reply:
Yeah, that's what I did initially. The negative margin was an ugly fix for what I thought was too large of a gap between the input and the predictions list:

<img width="433" alt="Screenshot 2024-03-26 at 1 08 44 PM" src="https://github.com/NCEAS/metacatui/assets/26600641/eabe98f9-82c8-4346-b9bd-29abe0b3626e">